airbyte-integrations/connectors/source-mysql/src/main/resources/spec.json

```diff
@@ -42,6 +42,13 @@
         "type": "string",
         "order": 5
       },
+      "ssl": {
+        "title": "SSL Connection",
+        "description": "Encrypt data using SSL.",
+        "type": "boolean",
+        "default": false,
```

Comment:
could you possibly set SSL as `true` by default?

Reply:
done